### PR TITLE
NERFC QPE processor added

### PR DESCRIPTION
**Pull request by jeffsuperglide**

### CODE REVIEW

**Comment by brettpalmberg** [comment]
@jeffsuperglide, I just did a git force-push after a rebase against `main`

File: src/tests/integration/default-test-suite/config.py [hunk 1]
```diff
@@ -74,6 +74,7 @@
     ("serfc-qpe-01h", "serfc-qpe-01h/xmrg0401202200z.grb.gz", None, None),
     ("serfc-qpf-06h", "serfc-qpf-06h/ALR_QPF_SFC_2022040100_006.grb.gz", None, None),
     ("wpc-qpf-2p5km", "wpc-qpf-2p5km/p06m_2022080912f168.grb", None, None),
+    ("nerfc-qpe-01h", "nerfc-qpe-01h/xmrg1207202215z.grb.gz", None, None),
```
Comment (brettpalmberg) [inline src/tests/integration/default-test-suite/config.py:77]: (picky) Please put this in alphabetical order.

**Review by brettpalmberg — CHANGES_REQUESTED** [review]

File: src/tests/integration/default-test-suite/config.py (hunk 1 above)
Comment (jeffsuperglide) [reply src/tests/integration/default-test-suite/config.py:77]: > (picky) Please put this in alphabetical order.

Did this on purpose 😬 

Not all in order.  Will change them all.

**Review by brettpalmberg — APPROVED** [review]

File: src/tests/integration/default-test-suite/config.py (hunk 1 above)
Comment (brettpalmberg) [reply src/tests/integration/default-test-suite/config.py]: Looks good! Thanks @jeffsuperglide.

**Pull request merged**